src/gentypes.jl

```diff
@@ -460,9 +456,7 @@ function generatetypes(
     root_name::Symbol = :Root,
 )
     # either a JSON.Array or JSON.Object
-    json = read_json_str(json_str)
-
-    # build a type for the JSON
+    json = read(json_str)
```

Comment:
wait, why are we not using `read_json_str` here anymore?

Reply:
`read` now does what `read_json_str` did, so it's redundant

Reply:
ah got it.